mobile/app.json

```
@@ -0,0 +1,32 @@
+{
+  "expo": {
+    "name": "makerspaceinventory",
+    "slug": "mobile",
+    "privacy": "public",
+    "sdkVersion": "32.0.0",
+    "platforms": [
+      "ios",
+      "android"
+    ],
+    "version": "1.0.0",
+    "orientation": "portrait",
+    "icon": "./logo.png",
+    "splash": {
+      "image": "./assets/splash.png",
```

Probably should be
```suggestion
      "image": "./assets/images/splash.png",
```
Current path raises error while running 'yarn start'.